fe/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java

```diff
@@ -104,9 +104,9 @@ public TxnStateListenerRegistry getListenerRegistry() {
         return listenerRegistry;
     }
 
-    public long beginTransaction(long dbId, String label, String coordinator, LoadJobSourceType sourceType)
-            throws AnalysisException, LabelAlreadyUsedException, BeginTransactionException {
-        return beginTransaction(dbId, label, -1, coordinator, sourceType, -1);
+    public long beginTransaction(long dbId, String label, String coordinator, LoadJobSourceType sourceType,
```

Comment:
Add a default timeout is more gentle.

Reply:
I just don't want to add another constructor, again...